socks5.py

```diff
@@ -331,8 +565,11 @@ def handle_udp(self, address, port):
             # remote-side socket
             ssock = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
             if CONNECT_HOST:
-                ssock.bind((CONNECT_HOST, 0))
-            logging.info('%s: udp association established', log_tag)
+                if "ipv4" in CONNECT_HOST:
+                    ssock.bind((CONNECT_HOST["ipv4"], 0))
+                elif "ipv6" in CONNECT_HOST:
+                    ssock.bind((CONNECT_HOST["ipv6"], 0))
```

Comment:
This doesn't work at all because the socket is created using `AF_INET` above. I haven't tested UDP stuff with the proxy lately; I'll probably look at trying HTTP3/QUIC connections through it to see if those work. There's a decent chance the old code doesn't handle UDP correctly...